mcux/drivers/lpc/CMakeLists.txt

```diff
@@ -22,3 +22,4 @@ zephyr_library_sources_ifdef(CONFIG_SOC_FLASH_MCUX		fsl_iap.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_MCUX_FLEXCOMM		fsl_spi.c fsl_flexcomm.c)
 zephyr_library_sources_ifdef(CONFIG_UART_MCUX_FLEXCOMM		fsl_usart.c fsl_flexcomm.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_MCUX_WWDT		fsl_wwdt.c)
+zephyr_library_sources_ifdef(CONFIG_PWM_MCUX_SCTIMER            fsl_sctimer.c)
```

Comment:
white space different from the rest of the file.

Reply:
Thanks @dleach02. I have made the change